src/fileinfo/file_information/file_information.h

```diff
@@ -23,637 +23,595 @@ namespace fileinfo {
-		void setResourceTable(const retdec::fileformat::ResourceTable *sTable);
-		void setStrings(const std::vector<retdec::fileformat::String> *sStrings);
-		void setCertificateTable(const retdec::fileformat::CertificateTable *sTable);
-		void setTlsInfo(const retdec::fileformat::TlsInfo *info);
-		void setSignatureVerified(bool verified);
-		void setLoadedBaseAddress(unsigned long long baseAddress);
-		void setLoaderStatusMessage(const std::string& statusMessage);
-		void setLoaderErrorInfo(const retdec::fileformat::LoaderErrorInfo & ldrErrInfo);
-		void setDotnetUsed(bool set);
-		void setDotnetRuntimeVersion(std::uint64_t majorVersion, std::uint64_t minorVersion);
-		void setDotnetMetadataHeaderAddress(std::uint64_t address);
-		void setDotnetMetadataStreamInfo(std::uint64_t streamOffset, std::uint64_t streamSize);
-		void setDotnetStringStreamInfo(std::uint64_t streamOffset, std::uint64_t streamSize);
-		void setDotnetBlobStreamInfo(std::uint64_t streamOffset, std::uint64_t streamSize);
-		void setDotnetGuidStreamInfo(std::uint64_t streamOffset, std::uint64_t streamSize);
-		void setDotnetUserStringStreamInfo(std::uint64_t streamOffset, std::uint64_t streamSize);
-		void setDotnetModuleVersionId(const std::string& moduleVersionId);
-		void setDotnetTypeLibId(const std::string& typeLibId);
-		void setDotnetDefinedClassList(const std::vector<std::shared_ptr<retdec::fileformat::DotnetClass>>& dotnetClassList);
-		void setDotnetImportedClassList(const std::vector<std::shared_ptr<retdec::fileformat::DotnetClass>>& dotnetClassList);
-		void setDotnetTypeRefhashCrc32(const std::string& crc32);
-		void setDotnetTypeRefhashMd5(const std::string& md5);
-		void setDotnetTypeRefhashSha256(const std::string& sha256);
-		void setAnomalies(const std::vector<std::pair<std::string,std::string>> &anom);
-		/// @}
-
-		/// @name Other methods
-		/// @{
-		void addFileFlagsDescriptor(std::string descriptor, std::string abbreviation);
-		void clearFileFlagsDescriptors();
-		void addDllFlagsDescriptor(std::string descriptor, std::string abbreviation);
-		void clearDllFlagsDescriptors();
-		void addDataDirectory(DataDirectory &dataDirectory);
-		void addSegment(FileSegment &fileSegment);
-		void addSection(FileSection &fileSection);
-		void addSymbolTable(SymbolTable &table);
-		void addRelocationTable(RelocationTable &table);
-		void addDynamicSection(DynamicSection &section);
-		void addElfNotes(ElfNotes &notes);
-		void addFileMapEntry(const FileMapEntry& entry);
-		void addAuxVectorEntry(const std::string& name, std::size_t value);
-		void addCryptoPattern(Pattern &pattern);
-		void removeRedundantCryptoRules();
-		void sortCryptoPatternMatches();
-		void addMalwarePattern(Pattern &pattern);
-		void sortMalwarePatternMatches();
-		void addOtherPattern(Pattern &pattern);
-		void sortOtherPatternMatches();
-		void addTool(retdec::cpdetect::DetectResult &tool);
-		void addLoadedSegment(const LoadedSegment& segment);
-		/// @}
+private:
```

Comment:
It is unfortunate to include this kind of change in a PR. It is needlessly convoluting things.